src/index.js

```diff
@@ -1,6 +1,19 @@
 const fibonacci = (n) => {
 
-  }
+  if (n<=1)  return [1];
 
+  let anterior = fibonacci(n - 1);
+  let longitudArray = anterior.length;
 
-  module.exports = fibonacci;
\ No newline at end of file
+  // Para que f(2) no genere un typo ponemos cero 
+  //por defecto a f(0) ya que no lo piden en el array
+  (!anterior[longitudArray - 2]) ? casilla2 = 0 : casilla2 = anterior[longitudArray - 2];
```

Comment:
use ternary conditional when you assign a value. For example:
```js
let lastNumber = last[arrayLenght - 2 ] ? last[arrayLenght - 2 ]  : 0;
```
if you will not use the assignment, use a `if else` block.